### DISCUSSION TITLE
How to add missing boxart?

### DISCUSSION THREAD

**Discussion by legolas119** [flyinghead/flycast#1407] [General]
hi, i have just configured Flycast. But there are 3 games (Gunlord, Neo XyX and fast striker) with missign boxart. I could add them adding the png file into the boxart folder, but i don't know which is the number i have to use to rename the .png file of the missing box...
can you help me please? thank you

**Comment by legolas119** [comment]
hi, anyone can help me ,please?

**Reply by flyinghead** [reply]
you can add your own images by editing the file `data\boxart\flycast-gamedb.json` (on Windows)

**Reply by legolas119** [reply]
yeah! thank you, it owkrs! many thanks!

**Reply by MesonW** [reply]
Just a quick note to say this works on Xbox as well as Windows. Though getting to the file is a bit more convoluted.
